.github/actions/label/build.js

```diff
@@ -100,12 +100,16 @@ const dispatchWorkflow = async ({ core, github }) => {
     head_sha: HEAD_SHA,
   });
 
-  if (workflow_runs.length != 1) {
+  if (workflow_runs.length === 0) {
     core.setFailed(`Incorrect workflow runs`);
     return;
   }
 
-  return workflow_runs[0];
+  let sorted_runs = workflow_runs.sort((a, b) => {
+    return new Date(b.created_at) - new Date(a.created_at);
+  })
+
+  return sorted_runs[0]
```

Comment:
The workflow in #3154 actually has been triggered:

https://github.com/gear-tech/gear/runs/16151228039

But for some unknown reasons, could be `close` + `open` + `label`, that github API could not find the proper place to queue the new created checks, so it failed to be displayed in the checks of the PR, the lines above provide a workaround for it, that if meet that kind of situation:

1. re-run the workflow to clean the previous status
2. re-label can trigger dispatch again